source/precalculus/source/05-EL/01.ptx

```diff
@@ -294,7 +294,7 @@
           <me>
             f(x)=ab^{x}
```

Comment:
Do you want a comma after the equation?

Reply:
I could go either way, but it probably does need one.

Reply:
I also don't have a preference....we should probably look at how we have been doing it in other sections to be consistent.